app/models/comment.rb

```diff
@@ -1,11 +1,11 @@
 class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post
   after_save :increment_post_comments_counter
 
   private
 
   def increment_post_comments_counter
-    post.increment!(:comments_counter)
+    post.increment!(:CommentsCounter)
   end
```

Comment:
- [ ]  Good job with test coverage 💪, but since we are aiming for 100% test coverage, and it is also a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md)for this project, "Add unit specs for all of your models' methods and validations", please add a unit test for `increment_post_comments_counter` method.